### ISSUE TITLE
Links on Origins map don't work

### ISSUE THREAD

**Issue by TaciteOFF** [openfoodfacts/openfoodfacts-server#175]
Hello,

On pages with origins of products from a category, such as :  http://world-fr.openfoodfacts.org/categorie/charcuteries/origines

If you click on countries on the map, links don't work.

**Comment by hangy** [comment]
It works on world.openfoodfacts.org or de.openfoodfacts.org, but not world-fr.openfoodfacts.org or de-en.openfoodfacts.org, because the click event handler uses the wrong domain:
```html
onRegionClick: function(e, code, region){
	if (countries_map_links[code]) {
		window.location.href = "http://world.openfoodfacts.org" + countries_map_links[code];
	}
  },
```

**Comment by hangy** [comment]
Fixed in 6af78b1 in production.
